examples/momento/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "momento-sequelize",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "basic": "tsc && node dist/basic.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "sequelize": "../../packages/core"
```

Comment:
this is still to be figured out

Reply:
i think this is fine for the github code for now, and we determined we can do a `pack` and give the tarball out to users for testing if they want to try it locally, yes?

Reply:
Yep correct